IPython/html/nbextensions.py

```diff
@@ -74,8 +77,10 @@ def _maybe_copy(src, dest, verbose=1):
     if _should_copy(src, dest, verbose):
         if verbose >= 1:
             print("copying %s -> %s" % (src, dest))
-        shutil.copy2(src, dest)
-
+        try:
+            shutil.copy2(src, dest)
+        except IOError as e:
+            print(str(e), file=sys.stderr)
```

Comment:
Fair enough.  Those weren't the case for my example extension, but I agree that those are the more likely cases.  I'll take out the error trapping.